Addon.Episerver.EnvironmentSynchronizer/Configuration/SiteDefinitionElement.cs

```diff
@@ -33,5 +33,11 @@ public HostCollection Hosts
 	        get => (HostCollection)base["hosts"];
         }
 
+        [ConfigurationProperty("Force", IsRequired = false)]
+        public bool Force
```

Comment:
What's the purpose of this configuration attribute? I can't see it being used in the code.

Reply:
I forgot to remove it. But an update is sent in with removal of Force settings.